src/cswinrt/strings/additions/Windows.Storage/WindowsRuntimeStorageExtensions.cs

```diff
@@ -12,14 +12,13 @@ namespace System.IO
     using global::Windows.Storage.Streams;
     using System.Runtime.InteropServices;
 
+#if !NETSTANDARD2_0
     /// <summary>
-    /// Contains extension methods that provide convenience helpers for WinRT IO.
+    /// Contains Net5 extension methods that provide convenience helpers for WinRT IO.
     /// </summary>
-    public static class WindowsRuntimeStorageExtensions
+    [global::System.Runtime.Versioning.SupportedOSPlatform("windows10.0.10240.0")]
+    public static class WindowsRuntimeStorageExtensionsNet5
```

Comment:
I wonder if this would be considered a API surface change for the Windows SDK projection.  Hopefully it won't as these are just extension APIs.

Reply:
good find - yes, this would be a breaking change for existing assemblies linking to the Windows projection.  I can certainly undo this organization to prevent that (and will)